modules/pytket-cirq/docs/changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 ~~~~~~~~~
 
+0.10.1 (May 2021)
+-----------------
+
+* Pinning cirq version to 0.10.
```

Comment:
Should this be 0.10.0?

Reply:
Because of the `~` this accepts any 0.10.x.

Reply:
KK